demo/dynamic_embedding/amazon-video-games-keras-eager/feature.py

```diff
@@ -12,6 +12,8 @@
 ]
 LABEL = 'verified_purchase'
 
+NUM_FEATURE_SLOTS = 0
```

Comment:
I believe opterating a global variable in class is good idea, that may cause conflict when there're many instances of the class, for example, your model have multi towers, they have different SLOTS.